pkg/appconsts/v2/app_consts.go

```diff
@@ -1,9 +1,7 @@
 package v2
 
 const (
-	Version              uint64 = 2
-	SquareSizeUpperBound int    = 128
-	SubtreeRootThreshold int    = 64
```

Comment:
1) these weren't being used 2) atm, I don't think we benefit from having copies of constants in each version. Perhaps we can version these differently? Just use the v1 version until they change, then specify the new version